test/input_gen/genModelTests_v2.py

```diff
@@ -590,7 +590,17 @@ def forward(self, inputs, labels):
         loss = self.loss(out, labels[0])
         return out, loss
 
+class UnsqueezeOperation(torch.nn.Module):
```

Comment:
Is it correct that you couldn't add the model unit test because of the input data type(integer type)? I haven't reviewed the model unit-test part yet. If possible, it would be great if you could work on modifying the model unit test to support integer types!

Reply:
@abhinav-nn I am also aware that the current `model unittest` has been designed to accept only float inputs. It is likely that the existing structure does not support integer inputs. Therefore, I would like to request an improvement to the `model unittest system` itself to support integer types! This task does not need to be completed within this PR; I would appreciate it if you could submit it as a separate PR.